sources/styles/app.css

```diff
@@ -1,58 +1,55 @@
-.webixappstart{
-    opacity:0;
+.user_block {
+	padding: 10px 0 0 55px;
 }
-.webixapp{
-    transition: opacity 500ms;
-    opacity: 1;
+
+.user_image {
+    width: 42px;
```

Comment:
mix of 2 spaces and 4 spaces in whole file

Reply:
Fixed